src/Symfony/Component/Lock/Tests/Store/FlockStoreTest.php

```diff
@@ -46,14 +46,23 @@ public function testConstructWhenRepositoryDoesNotExist()
     public function testConstructWhenRepositoryIsNotWriteable()
     {
         $this->expectException('Symfony\Component\Lock\Exception\InvalidArgumentException');
-        $this->expectExceptionMessage('The directory "/" is not writable.');
+        $this->expectExceptionMessage('The FlockStore directory "/" is not writable.');
         if (!getenv('USER') || 'root' === getenv('USER')) {
             $this->markTestSkipped('This test will fail if run under superuser');
         }
 
         new FlockStore('/');
     }
 
+    public function testConstructWithSubdir()
+    {
+        if (!getenv('USER') || 'root' === getenv('USER')) {
+            $this->markTestSkipped('This test will fail if run under superuser');
+        }
+
+        new FlockStore(sys_get_temp_dir().'/sf-flock');
```

Comment:
this test has no assertion, which marks it as risky

Reply:
It's worth than that, it makes our CI fails currently :)

Reply:
i'm on it